app/src/main/java/org/mozilla/fenix/search/SearchFragment.kt

```diff
@@ -94,7 +94,10 @@ class SearchFragment : Fragment(), BackHandler {
             ) {
                 ToolbarViewModel(SearchState(url, session?.searchTerms ?: "", isEditing = true))
             }
-        )
+        ).also {
+            // Remove background from toolbar view since it conflicts with the search UI.
+            it.uiView.view.background = null
```

Comment:
In general, it'd be better to make UI changes in the ToolbarUIView. But I guess this is okay.

Reply:
That was my original intention, but I couldn't figure out how to do that in the context of the search fragment. Happy to update the PR to the right way!

Reply:
Yeah, @colintheshots for context, we use the same component and UIView for Search and Browser Fragment, but only one needs a background.